playground/nextjs-app-router/components/Demo.tsx

```diff
@@ -159,7 +119,7 @@ function Demo() {
       </div>
       <div className="linear-gradient(to_bottom,#f0f0f0_1px,transparent_1px)] flex flex-1 flex-col bg-[linear-gradient(to_right,#f0f0f0_1px,transparent_1px), bg-[size:6rem_4rem]">
         <div className="flex h-full w-full flex-col items-center justify-center">
-          {renderActiveComponent()}
+          {ActiveComponent && <ActiveComponent />}
```

Comment:
since `ActiveComponent` maps to a component or `null`, do you need the logic here?